content/security/docs/incidents.md

```diff
@@ -33,6 +33,15 @@ kubectl get pods -o json --namespace <namespace> | \
     "\(.metadata.name) \(.spec.nodeName)"'
 ```
 
+### Identify the offending Pods and worker nodes using container image
```

Comment:
I think you need to specify which images, i.e. images of containers that have a CVE that have been exploited. When It is not clear what "offending Pods" are in line 36.

Reply:
Added text as suggested in this commit.

Reply:
I think you're still missing with word "compromised", e.g. "Identify the offending Pods and worker nodes that are running a container with a known vulnerability or compromise."